packages/forms/stories-core/json/fields/core-timezoneList.json

```diff
@@ -0,0 +1,24 @@
+{
+	"doc": "TimezoneList/README.md",
+	"jsonSchema": {
+		"type": "object",
+		"properties": {
+			"timezone": {
+				"type": "string"
```

Comment:
It could be useful to be able to specify `"type": "object"` and to get the selected timezone's full object (timezone, offset ...). Other way around is to expose timezone helpers.
There's a use case in TMC where the full object is managed in the form submit (might be the only use case though).

Reply:
In the end, `@talend/utils` (which is already a dep needed for this widget) provides a way to get timezone offset from its name.